### DISCUSSION TITLE
No entiendo el error de esto, Ejercicio 7 "Numeros Primos"

### DISCUSSION THREAD

**Discussion by RainfallZen** [INGCOM-UNRN-P1/cursada-2023#55] [Preguntas y respuestas] [answered]
Hola, buenas queria ver si podian ayudarme a solucionar mi problema con la terminal ya que a la hora de hacer el gcc a dicho archivo me tira un error que me dice que el problema esta en el main, pero a la hora de revisarlo creo que yo que esta bien.

(https://github.com/INGCOM-UNRN-P1/tp1-2023-RainfallZen/blob/main/ejercicio7.c)

Y el mensaje del compilador era

<img width="960" alt="url" src="https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/127630e0-a9b5-4e7d-bbb2-bff030518eb6">

**Comment by RainfallZen** [comment]
Capaz es un error tonto, pero no logro verlo.

**Comment by TeoInMaghine** [comment]
Hola, @martinvilu supongo que tiene acceso al repositorio que mandaste, pero si querés que otros estudiantes lo puedan ver podés copiar el código directamente en el posteo ¿Qué mensaje de error te tira?

**Reply by martinvilu** [reply, marked as answer]
Este era el error; un `\` al revés.
```diff
-printf("Ingrese el valor:n\");
+printf("Ingrese el valor:\n");
```

**Reply by martinvilu** [reply]
Gracias @TeoInMaghine por el recordatorio, así la respuesta quedó completa
